.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this file should be removed from the commit

Reply:
Removed